interfaces/Libraries.ts

```diff
@@ -184,6 +184,34 @@ nlp = spacy.load("${nameWithoutNamespace(model.modelId)}")
 import ${nameWithoutNamespace(model.modelId)}
 nlp = ${nameWithoutNamespace(model.modelId)}.load()`;
 
+const speechbrainASR = (model: ModelData) =>
+`from speechbrain.pretrained import EncoderDecoderASR
+
+asr_model = EncoderDecoderASR.from_hparams(source="${model.modelId}")
+asr_model.transcribe_file("file.wav")`;
+
+const speechbrainEnhancement = (model: ModelData) =>
+`from speechbrain.pretrained import SpectralMaskEnhancement
+
+enhance_model = SpectralMaskEnhancement.from_hparams(source="${model.modelId}")
+enhanced = enhance_model.enhance_file("file.wav")`;
+
+const speechbrainSeparator = (model: ModelData) =>
+`from speechbrain.pretrained import SepformerSeparation
+
+separator_model = SepformerSeparation.from_hparams(source="${model.modelId}")
+est_sources = separator_model.separate_file("file.wav")`;
+
+const speechbrain = (model: ModelData) => {
+	if (model.tags.includes("automatic-speech-recognition")){
+		return speechbrainASR(model);
+	} else if (model.tags.includes("speech-enhancement")) {
```

Comment:
what about defining a new, generic, `audio-to-audio`?

pipeline types should try to stay kinda limited in number IMO, so rather generic (see `token-classification` for text for instance as opposed to ner, pos-tagging, etc.)

Reply:
I think a single, generic type would be nice. 

On the other hand, the code snippet needs to look different for the different types. One potential solution would be to expect from the framework side to unify the syntax for these different types so a single code snippet/inference API would work for all of them (similar to what is happening with TensorFlowTTS). I don't know if this is something we can ask though :)

Reply:
After discussing with Nicolas, we decided to keep as it was. Although an `audio-to-audio` tag would be ideal, we ¡ still need more granular tags in order to know which classes/methods to use.

Reply:
Then let's add both. On the website side we won't expose that many tasks (in the filtering UX etc) so otherwise those models won't be discoverable from the website